supertokens_python/recipe/emailpassword/asyncio/__init__.py

```diff
@@ -37,22 +41,32 @@ async def is_email_verified(user_id: str):
         user_id, email)
 
 
+@deprecated(reason="Use supertankers.get_user_oldest_first(...) function instead IF using core version >= 3.5")
 async def get_users_oldest_first(limit: int = None, next_pagination: str = None):
     return await EmailPasswordRecipe.get_instance().recipe_implementation.get_users_oldest_first(limit, next_pagination)
 
 
+@deprecated(reason="Use supertankers.get_users_newest_first(...) function instead IF using core version >= 3.5")
 async def get_users_newest_first(limit: int = None, next_pagination: str = None):
     return await EmailPasswordRecipe.get_instance().recipe_implementation.get_users_newest_first(limit, next_pagination)
 
 
+@deprecated(reason="Use supertankers.get_user_count(...) function instead IF using core version >= 3.5")
 async def get_user_count():
     return await EmailPasswordRecipe.get_instance().recipe_implementation.get_user_count()
 
 
+async def update_email_or_password(user_id: str, email: Union[str, None] = None,
+                                   password: Union[str, None] = None):
+    return await EmailPasswordRecipe.get_instance().recipe_implementation.update_email_or_password(user_id, email,
+                                                                                                   password)
+
+
 async def get_user_by_id(user_id: str):
     return await EmailPasswordRecipe.get_instance().recipe_implementation.get_user_by_id(user_id)
 
 
+@deprecated(reason="Use supertankers.get_user_by_email(...) function instead IF using core version >= 3.5")
```

Comment:
why this deprecation warning? We discussed on the call that this is not deprecated!